BuzzerWolf.Server/appsettings.Development.json.sample

```diff
@@ -11,5 +11,9 @@
   "BBAPIPublicUser": {
     "UserName": "",
     "AccessKey": ""
+  },
+  "NextjsStaticHosting": {
```

Comment:
this should be automagic and never need to be changed, but I'd say it's good to know it's here